docs/sources/panels-visualizations/visualizations/bar-chart/index.md

```diff
@@ -73,6 +85,18 @@ Controls the width of groups. 1 = max with, 0 = min width.
 
 Controls the width of bars. 1 = Max width, 0 = Min width.
 
+### Bar radius
+
+Controls the radius of the bars. 0.5 = Max radius, 0 = Min radius.
+
+### Highlight full area on cover
+
+Controls if the full surrounding area of the bar is highlighted when hovering over the bar.
+
+### Color by field
+
+Controls whether the bars are colored by a sibling field.
```

Comment:
I looked into providing an example and possibly discovered a bug. My understanding of this was that you could color the bars representing some values by the color of a different field in the dataset. The more I think about it the more confusing I find it aswell since the legend seems to show wrong values when this setting is set.

I have removed the doc for this option and will look into it's functionality better. Will probably come back with a new docs PR when I figure if there are bugs or my understanding is wrong of it.

Reply:
you can choose another field to use for coloring the bars. that field can be something like "status" strings: "pass", "fail". then you set up value mappings (or thresholds) on that field to control the colors.

https://play.grafana.org/d/barchart-thresholds-mappings/barchart-thresholds-and-mappings?orgId=1

yes, the legend might need some more work.

Reply:
see original PR for more use cases "build status" bars. https://github.com/grafana/grafana/pull/43257

Reply:
Okay, I understand. Then I guess we should actually add it in the docs... maybe something like `Color bars based on another field's values.` And possibly add this PASS/FAIL with value mappings example in there somehow?